test/integration/sanity/multi-case-url.test.js

```diff
@@ -0,0 +1,117 @@
+var expect = require('chai').expect;
+describe('Different case url', function() {
+
+    describe('Lowercase', function() {
+        var testrun;
+        before(function(done) {
+            this.run({
+                collection: {
+                    item: [{
+                        request: {
+                            url: 'http://postman-echo.com/post',
+                            method: 'POST',
+                            body: {
+                                mode: 'raw',
+                                raw: ''
+                            }
+                        }
+                    }]
+                }
+            }, function(err, result) {
+                testrun = result;
+                done(err);
+            });
+        });
+
+        it('Should process request when url is in lowercase', function() {
+            expect(testrun.response.getCall(0).args[2]).to.have.property('code', 200);
+            expect(testrun.response.getCall(0).args[3].url.toString()).eql('http://postman-echo.com/post');
+        });
+    });
+
+    describe('Uppercase', function() {
+        var testrun;
+        before(function(done) {
+            this.run({
+                collection: {
+                    item: [{
+                        request: {
+                            url: 'HTTP://POSTMAN-ECHO.COM/POST',
+                            method: 'POST',
+                            body: {
+                                mode: 'raw',
+                                raw: ''
+                            }
+                        }
+                    }]
+                }
+            }, function(err, result) {
+                testrun = result;
+                done(err);
+            });
+        });
+
+        it('Should process request when url is in uppercase', function() {
```

Comment:
Required sinon assertions for all the tests to make sure that run is executed without any error.